fix(css): Improve contrast for angle

rohini-ranjanR opened the pull request:
<!-- 🙌 Thanks for contributing! Adding details below will help us to merge your PR faster. -->

### Description
improve contrast by changing color for angle issue #2054 
<!-- ✍️ Summarize your changes in one or two sentences -->

### Motivation

<!-- ❓ Why are you making these changes and how do they help? -->

### Additional details

<!-- 🔗 Link to documentation, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests
Relates #2054 
<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->

rohini-ranjanR committed:
improve contrast for angle

improve contrast by changing color.

NiedziolkaMichal commented on live-examples/css-examples/values-and-units/angle.css:
```diff
@@ -1,5 +1,5 @@
 #example-element {
-    background-color: lightsalmon;
+    background-color: #ff5b9b;
```
This color still doesn't meet the minimum contrast guidelines, as the text to background contrast ratio is `2.91`.
How about the #0454aa - which meets [AAA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast#related_wcag_success_criteria) requirement with score `7.36`?

rohini-ranjanR replied:
I think this color looks good in dark theme but when we switch to light the text in black will little hard to look. What about #ffb703 it looks good in both dark and light theme.

NiedziolkaMichal replied:
This color has a contrast ratio of `1.75` and even for me it's difficult to read the content:
![image](https://user-images.githubusercontent.com/100634371/219430242-81d79d78-d12a-4741-b1da-d40213e9b05e.png)

rohini-ranjanR replied:
ok suggested color applied

NiedziolkaMichal replied:
@rohini-ranjanR We could still go with something else that you think would be better for both light and dark themes. To check the contrast ratio, first, inspect `#example-element` in dev tools, then add `color: white;` declaration in it(this shouldn't have any effect), and now click on the `white` dot:
![image](https://user-images.githubusercontent.com/100634371/219432546-35fc9093-1d8a-4c83-9689-c4f45970d49c.png)

The contrast ratio will be displayed at the bottom, comparing color with `background-color`.

rohini-ranjanR replied:
Thanks @NiedziolkaMichal . I am using #0118f3 here that is giving a contrast of `8.65`

![image](https://user-images.githubusercontent.com/109434545/219447990-60bfa5fb-4ac0-4bc6-a2f3-76e862055244.png)

looks like this in **dark theme** 

![image](https://user-images.githubusercontent.com/109434545/219448278-bb5acea0-cdf2-4fea-8005-491051ec1c69.png)

and this in **light theme** 
![image](https://user-images.githubusercontent.com/109434545/219448117-2fb66f93-1f2e-4f26-bacc-9447be727311.png)

NiedziolkaMichal replied:
Thanks for noticing. How about also changing the color to white, so the example looks good in both themes?

rohini-ranjanR replied:
okay, thanks for your suggestions.

rohini-ranjanR committed:
apply suggested color

rohini-ranjanR committed:
change color

rohini-ranjanR committed:
change text color

rohini-ranjanR committed:
fix ESLint error

fixing ESLint error take help from https://blog.devgenius.io/what-are-eslint-rules-d2d99bf5387

rohini-ranjanR committed:
restoring values

rohini-ranjanR committed:
fix eslint

NiedziolkaMichal reviewed (approved):
It looks good. Thank you for your contribution.
